src/Temporalio/Worker/WorkflowInstance.cs

```diff
@@ -538,8 +538,25 @@ public WorkflowActivationCompletion Activate(WorkflowActivation act)
                     {
                         // We must set the sync context to null so work isn't posted there
                         SynchronizationContext.SetSynchronizationContext(null);
+                        // TODO: Temporary workaround in lieu of https://github.com/temporalio/sdk-dotnet/issues/375
+                        var sortedJobs = act.Jobs.OrderBy(j =>
```

Comment:
I am a bit surprised we didn't order before. But I assume this order by matches previous Core versions before this update? Meaning we feel fairly confident that replays will work the same on .NET SDK upgrade here?

Reply:
Yes, this doesn't change anything. It matches the previous Core ordering